Formula/w/wireshark.rb

```diff
@@ -8,9 +8,11 @@ class Wireshark < Formula
   revision 1
   head "https://gitlab.com/wireshark/wireshark.git", branch: "master"
 
+  # Upstream indicates stable releases with an even-numbered minor (see:
+  # https://wiki.wireshark.org/Development/ReleaseNumbers).
```

Comment:
are they still following this, looks like the last update was 2020 Aug?
![image](https://github.com/Homebrew/homebrew-core/assets/1580956/55299821-edab-43d4-b82c-c42b3e7b79ac)

Reply:
> are they still following this, looks like the last update was 2020 Aug?

Yes, if you look back at snapshots of the download page to see which versions are listed as the "Development Release", they continue to follow the outlined version scheme.

If you're referring to the versions that are listed in that screenshot as being old, those are just for example. The Release Number page doesn't contain any content that needs to be regularly kept up to date, so the date isn't meaningful.